### PR TITLE
Update the generate_false_positive.py script and false positives list.

### PR DESCRIPTION
**Pull request by Szarad**
Update generate_false_positive.py script to be up to date with current NLP API code. Run the script to update list of false positives.

### CODE REVIEW

File: bin/generate_false_positive.py [hunk 1]
```diff
@@ -73,32 +73,6 @@ def check_if_server_is_running(api_url):
 for file in os.listdir(base_directory):
     training_data_paths.append(base_directory + file)
 
-
```
Comment (lsmith77) [inline bin/generate_false_positive.py:76]: why did you remove these?

Comment (Szarad) [reply bin/generate_false_positive.py:76]: These lines weren't used, so to make the code more readable.

File: bin/generate_false_positive.py [hunk 2]
```diff
@@ -130,7 +104,7 @@ def check_if_server_is_running(api_url):
     if "~" in word:
         for german_gender_ending in endings:
             alternative = ResultOut.getGenderedRolesFormatInclusive(
-                word, german_gender_ending
+                german_gender_ending, word
```
Comment (lsmith77) [inline bin/generate_false_positive.py:107]: ooops .. good catch
